main.cpp

```diff
@@ -23,6 +23,17 @@ namespace TelCoColorCoder
         std::cout << "Got pair number " << pairNumber << std::endl;
         assert(pairNumber == expectedPairNumber);
     }
+
+    std::string ColorCoding()
```

Comment:
This function is intended for a consumer who wants to print the color codes (it's production code). Rest of this file has test code, which only proves that our code is working. It's not meant to be used by a consumer.
By putting production code together with test code, we are forcing people to take the test code when they only want to use the function that forms the reference manual